docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -226,15 +226,15 @@ The nodes should be configured to allow key based SSH connections as user ``root
 
 .. _client-acceptance-tests:
```

Comment:
:question: Perhaps this page should be merged somehow with http://doc-dev.clusterhq.com/gettinginvolved/client-testing.html?highlight=docker